### PR TITLE
Add new layer definitions for the IdealVGI project

### PR DESCRIPTION
**Pull request by matthiasschaub**
### Corresponding issue
Closes #134 

### Checklist
- [x] I have updated my branch to `main` (e.g. through `git rebase main`)
- [x] My code follows the [style guide](https://github.com/GIScience/ohsome-quality-analyst/blob/main/CONTRIBUTING.md#style-guide) and was checked with [pre-commit](https://github.com/GIScience/ohsome-quality-analyst/blob/main/CONTRIBUTING.md#pre-commit) before committing
~~- [ ] I have commented my code~~
~~- [ ] I have added sufficient unit and integration [tests](https://github.com/GIScience/ohsome-quality-analyst/blob/main/docs/development_setup.md#tests)~~
- [x] I have updated the [CHANGELOG.md](https://github.com/GIScience/ohsome-quality-analyst/blob/main/CHANGELOG.md)

### CODE REVIEW

**Review by joker234 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -28,6 +28,7 @@
 - Minor improvements to the metadata and docstrings of indicators ([#110])
 - Change return type from bool to None for indicator for functions: preprocess, calculate, create_figure ([#96])
 - Implement mapping of custom feature id to default (unique) feature id ([#83])
+- Add new layer definitions for the IdealVGI project ([134])
```
[the remark this answers is not in the document]
Comment (joker234) [reply CHANGELOG.md:31]: Is this a new feature or an other change?

**Review by joker234 — APPROVED** [review]

**Review by joker234 — APPROVED** [review]

File: workers/ohsome_quality_analyst/ohsome/layer_definitions.yaml [hunk 2]
```diff
@@ -236,3 +236,22 @@ mapaction_rivers_length:
...
+  name: IDEAL-VGI Infrastructure
+  description: Objects related to infrastructure
+  endpoint: elements/length
+  filter: (aerialway=* or aeroway=* or highway=* or power=* or railway=* or telecom=*) and geometry:line
+
+ideal_vgi_poi:
+  name: IDEAL-VGI Points-of-Interests
+  description: Features related to Points-of-Interests
+  endpoint: elements/count
+  filter: (amenity=* or craft=* or emergency=* or historic=* or leisure=* or man_made=* or office=* or place=* or shop=* or tourism=*) and geometry:point
```
Comment (SlowMo24) [inline workers/ohsome_quality_analyst/ohsome/layer_definitions.yaml:250]: By limiting this type to points some elements will be missed which I don't see mentioned in the documentation. I would not include relations but for Heidelberg you miss about 10-20% of elements: https://api.ohsome.org/v1/elements/count/groupBy/type?bboxes=8.67%2C49.39%2C8.71%2C49.42&filter=amenity%3D*%20or%20craft%3D*%20or%20emergency%3D*%20or%20historic%3D*%20or%20leisure%3D*%20or%20man_made%3D*%20or%20office%3D*%20or%20place%3D*%20or%20shop%3D*%20or%20tourism%3D*&format=json&time=2021-08-01

```
{
  "attribution": {
    "url": "https://ohsome.org/copyrights",
    "text": "© OpenStreetMap contributors"
  },
  "apiVersion": "1.6.0",
  "groupByResult": [
    {
      "groupByObject": "NODE",
      "result": [
        {
          "timestamp": "2021-08-01T00:00:00Z",
          "value": 4862
        }
      ]
    },
    {
      "groupByObject": "WAY",
      "result": [
        {
          "timestamp": "2021-08-01T00:00:00Z",
          "value": 941
        }
      ]
    },
    {
      "groupByObject": "RELATION",
      "result": [
        {
          "timestamp": "2021-08-01T00:00:00Z",
          "value": 24
        }
      ]
    }
  ]
}
```

File: workers/ohsome_quality_analyst/ohsome/layer_definitions.yaml [hunk 3]
```diff
@@ -236,3 +236,22 @@ mapaction_rivers_length:
     Length of objects identified as rivers (or riverbanks)
   endpoint: elements/length
   filter: waterway in (riverbank,river) and type:way
+
+ideal_vgi_infrastructure:
```
Comment (SlowMo24) [inline workers/ohsome_quality_analyst/ohsome/layer_definitions.yaml:240]: If I remember correctly Alex specifically requested to drop project-specific naming, if possible. We are happy to incorporate any definition of 'infrastructure' you come up with, as long as it is superior to this one, right Leonie? If you want to have separate infrastructure definitions I would suggest something like `infrastructure-full` with this definition and then e.g. `infrastructure-transportation` with only _highways_ and _railways_.

File: workers/ohsome_quality_analyst/ohsome/layer_definitions.yaml (hunk 2 above)
Comment (tyrasd) [reply workers/ohsome_quality_analyst/ohsome/layer_definitions.yaml:250]: +1 for not restricting to just nodes. I'd say that one could extend the request to polygons as well, i.e. `… and (geometry:point or geometry:polygon)`

Comment (joker234) [reply workers/ohsome_quality_analyst/ohsome/layer_definitions.yaml:250]: You're correct, it shouldn't only contain `geometry:point`. I would suggest `(geometry:point or geometry:polygon)`. I checked `geometry:line` as well, but this includes a lot of features that are imho no POIs like these:
https://www.openstreetmap.org/way/26088966
https://www.openstreetmap.org/way/158025770
https://www.openstreetmap.org/way/312561922
https://www.openstreetmap.org/way/196967009

I would suggest:
 * use these `point` and `polygon` geometries
 * remove `emergency` key, as it isn't a key defining the primary use of a feature, mostly used on roads
 * remove `place` key as it denotes settlements, not POIs
 * the `man_made` key includes a lot of non-POI features like bridges, masts, embankments, …
 * think how to deal with the `amenity` key as this includes a lot of non-POI features

To get an idea of the complexity of this issue you can have a look at the [POI display OSM Wiki](https://wiki.openstreetmap.org/wiki/POI_display) page.

Comment (joker234) [reply workers/ohsome_quality_analyst/ohsome/layer_definitions.yaml:250]: created #161 for further discussion

**Pull request merged**